sormas-api/src/main/java/de/symeda/sormas/api/contact/ContactFacade.java

```diff
@@ -144,7 +145,10 @@ List<ContactFollowUpDto> getContactFollowUpList(
 
 	void deleteContactAsDuplicate(String uuid, String duplicateOfUuid);
 
-	List<MergeContactIndexDto[]> getContactsForDuplicateMerging(ContactCriteria criteria, boolean showDuplicatesWithDifferentRegion);
+	List<MergeContactIndexDto[]> getContactsForDuplicateMerging(
```

Comment:
I first wanted to ask you to replicate what we have done in CaseFacade.getCasesForDuplicateMerging (not null and upper limit), but thinking about it again, I'm in favor of giving however has access to the backend the power to query for as many duplicates as they want.
To have this consistent, please adjust the CaseFacade.getCasesForDuplicateMerging accordingly. This means the constants can be moved to the UI. See my other comment.